main/upower/APKBUILD

```diff
@@ -1,17 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=upower
-pkgver=0.99.7
+pkgver=0.99.10
+_distver="$(echo ${pkgver} | tr '.' '_')"
 pkgrel=0
 pkgdesc="Power Management Services"
 url="https://upower.freedesktop.org"
 arch="all"
 license="GPL-2.0-or-later"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev
-	dbus-glib-dev libxslt gobject-introspection-dev docbook-xsl"
-source="https://upower.freedesktop.org/releases/upower-$pkgver.tar.xz"
-# XXX: tests are broken on alpine for some reason, but not on adelie
-options="!check"
+makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev gtk-doc
```

Comment:
I am curious, what is `gtk-doc` for here?

Reply:
Required for autogen.sh because we don't have a gtk-doc-am package or similar from what I can see

Reply:
Interesting.